modules/load-images/load-images.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
```

Comment:
Wouldn't it be better to rename the folder "load-images" to something similar to "scripts"?